package.json

```diff
@@ -1,14 +1,15 @@
 {
   "private": true,
   "scripts": {
+    "build": "nx run-many --target=build --no-private",
     "bootstrap": "lerna bootstrap --force-local",
-    "build": "lerna run build --no-private",
     "rebuild": "npm run clean && npm clean-install && npm run build",
+    "test": "nx test @taquito/taquito --collectCoverage",
```

Comment:
Curious to know why the script "test" is changed to only run unit tests from the @taquito/taquito package rather than all packages.

Reply:
I thought this thing recursed, and that I was giving the required (top-level) project name. So this needs to be modified.

Reply:
This has been resolved in the latest version of the branch.